pyracf/common/improper_userid_error.py

```diff
@@ -0,0 +1,17 @@
+"""Exception to use when attempting to set an invalid userid for pyRACF to run under."""
+
+
+class ImproperUserIDError(Exception):
```

Comment:
I suggest changing the name of this exception to `UserIdError`.

As for making the letter 'd' in 'ID' lower case, there seems to be some disagreement in the community on how acronyms and more specifically two letter acronyms should be handled in camel/pascal case. Google's JavaScript Style Guide and a text to pascal case converter that I found, seem to indicate that all acronyms should follow the pascal case conventions like all other words. I am inclined to go with that standard so that there is no confusion.

https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined
https://www.atqed.com/string-pascal-case

Reply:
That is a reasonable direction to go with it. I will make sure that is changed in the necessary spots.